moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast.h

```diff
@@ -36,8 +36,7 @@
 #include <list>
 #include <stdexcept>
 
-#ifndef IKFAST_HEADER_COMMON
-#define IKFAST_HEADER_COMMON
+#pragma once
```

Comment:
It looks like you forgot to remove the corresponding `#endif` in this file

Reply:
Oh, I was trying to figure how it works but forgot to ask when committed, as I removed #ifndef and #define for IKFAST_HEADER_COMMON, I thought I am supposed to remove #endif for IKFAST_HEADER_COMMON only and it didn't exist in that file, and only #endif was for IKFAST_HAS_LIBRARY. Just removed it, committing now.

Reply:
there is no #endif for IKFAST_HEADER_COMMON though? Or am I misreading the files

Reply:
Fixed it, sorry i am dumb lol.
Btw, I am getting error while trying to push origin, "failed to push refs to <repo-link>", it wants me to git pull something but what should I git pull? (again sorry for dumb questions)

Reply:
That's because I pushed some commits to fix the clang-format issue.  If you run git pull locally it'll do the merge of the commits I added with yours and then you should be able to push.

Reply:
yeah, but running git pull locally tells me there is no tracking info for current branch, and asks me to specify the branch i want to merge with.

Reply:
You have to either set the tracking or use the full git pull command.

git pull [remote, probably origin] [your branch name]